src/main/java/com/cdancy/bitbucket/rest/features/BranchApi.java

```diff
@@ -126,6 +127,23 @@ BranchModel model(@PathParam("project") String project,
     BranchModelConfiguration getModelConfiguration(@PathParam("project") String project,
                                                    @PathParam("repo") String repo);
 
+    @Named("branch:get-model-configuration")
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Path("/branch-utils/{jclouds.api-version}/projects/{project}/repos/{repo}/branchmodel/configuration")
+    @Fallback(BitbucketFallbacks.BranchModelConfigurationOnError.class)
+    @PUT
+    BranchModelConfiguration updateModelConfiguration(@PathParam("project") String project,
+                                                      @PathParam("repo") String repo,
+                                                      @BinderParam(BindToJsonPayload.class) CreateBranchModelConfiguration config);
+
+    @Named("branch:delete-model-configuration")
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Path("/branch-utils/{jclouds.api-version}/projects/{project}/repos/{repo}/branchmodel/configuration")
+    @Fallback(BitbucketFallbacks.FalseOnError.class)
+    @DELETE
+    boolean deleteModelConfiguration(@PathParam("project") String project,
```

Comment:
Yup, this endpoint isn't in documentation.

Source : https://jira.atlassian.com/browse/BSERV-5411